packages/PoppyPrint-Core.package/PPFormatterConfig.class/class/getClassMethodsWithPragma.inClasses..st

```diff
@@ -0,0 +1,9 @@
+helpers
+getClassMethodsWithPragma: aSymbol inClasses: aClassList
+
+	| stream methodsWithPragma |
+	stream := WriteStream with: #().
+	(aClassList collect: [:methodClass | Pragma allNamed: aSymbol in: (methodClass class)]) do: [:listOfMethods | stream nextPutAll: listOfMethods].
+	methodsWithPragma := stream contents.
+	methodsWithPragma := methodsWithPragma collect: [:aPragma | aPragma method].
+	^ methodsWithPragma
```

Comment:
FYIO, I think you could also write this a bit shorter (not tested :)):

```suggestion
	^ (aClassList gather: [:methodClass | Pragma allNamed: aSymbol in: methodClass class])
		collect: [:pragma | pragma method]
```

Reply:
Thats a good Idea. I did not know that gasther flattens a 2D Array